md/release-notes.md

```diff
@@ -1,24 +1,100 @@
 # Release notes
 
 ::: info
-**Note:** The 7.7 is currently work in progress (wip). The 7.7.0 GA is planned on June 2018.
+**Note:** Bonita 7.7.0 is currently work in progress (wip). Beta version is due 17th May and GA version is planned 7th June 2018.
 :::
 
 
+## New values added
+
+<a id="git"/>
+
+### Collaborative development with Git (starting with Teamwork edition)
+Working as a team on a Bonita project, securing resources on a shared server, and keeping track of the resources revisions is now possible with the integration of Eclipse [EGit plug-in](http://wiki.eclipse.org/EGit/User_Guide) in Bonita Studio, as well as the creation of a dedicated gitignore file.  
+Managing branches, viewing the differences between two revisions of a file, or between the local and remote repositories, as well as managing conflicts for a given file can be performed. Find more in the dedicated [documentation page section](workspaces-and-repositories.md#git) and in the howto: [Share a repository on GitHub](share-a-repository-on-github.md).
+
+<a id="bdm-access-control"/>
+
+### Business data access control (BDM AC) (starting with Efficiency edition)
+It is now possible to protect the access to BDM data through REST API calls.  
+This feature, [**BDM access control**](bdm-access-control.md), is available in Bonita Studio and works as a **white list** mechanism: through access rules, access is granted to entire business objects or to some attributes only, so that only users mapped with the selected **profiles** can access the data. 
```

Comment:
`available in Bonita Studio` => The feature is part of the product, and the fact that it is present in Bonita Studio is just because the User interface makes it more visible.

Reply:
Technically, you are right.
"In the Studio" means that this is in the Studio UI that Dave needs to look for it.